README.md

```diff
@@ -11,34 +11,24 @@ Other sources include (in order of perceived helpfulness):
 * https://wiki.gentoo.org/wiki/Trusted_Boot
 * https://fedoraproject.org/wiki/Tboot
 
-## General procedure
-
-1. Enable and take ownership of the TPM
-2. Download and install SINIT
-3. Determine kernel parameters for linux and tboot
-    1. `intel_iommu=on` must be set for linux
-    2. There may only ever be one space between kernel parameters
-4. Create policy
-    1. Policy must be regenerated every time there is a new kernel or the kernel parameters are changed
-5. Create the `tboot` boot entry
-6. Verify that the SINIT is the last module in the `tboot` boot entry
-7. Reboot
-
 ## Preparation
 
 1. Activate the TPM in the BIOS and set a BIOS password
 2. Ensure VTd is enabled in the BIOS
 3. Boot into the kernel you intend to trust
 4. Install trousers, tpm-tools, tboot and start the tcsd daemon:
-    `yum install -y trousers tpm-tools tboot; systemctl start tcsd`
-5. Own the TPM using the well-known SRK password:
-    `tpm_takeownership -z`
-6. Download the appropriate SINIT for your platform:
-    `https://software.intel.com/en-us/articles/intel-trusted-execution-technology`
+    `yum install -y trousers tpm-tools tboot; systemctl start tcsd; chkconfig tcsd on`
```

Comment:
> chkconfig tcsd on

the systemd way to do this is `systemctl enable tcsd` and if you're trying to be backwards compatible `systemctl start tcsd` should be a `service` call